### PR TITLE
Irj_file_validation

### PR DESCRIPTION
**Pull request by mdurero**
Short term plan: add a small standalone module for IRJ file validation.

Long term: allow to validate and translate multiple file formats (APIs, IRJ, Alpage…)

For the IRJ validation, I have covered all cases found in 2022 files. For completion sake I would have liked to test for #CONTROLES-RAPPELS part, but it creates a conflict and I do not understand why.

Note: this error handling is deprecated in Menhir.

### CODE REVIEW

**Comment by mdurero** [comment]
@denismerigoux AFAIK, should add some tests for the new module in the CI, otherwise in itself it seems OK for a first merge.

File: dune-project [hunk 1]
```diff
@@ -7,11 +7,11 @@
 (generate_opam_files true)
 
 (source
- (uri git+https://gitlab.inria.fr/verifisc/mlang.git))
+ (uri git+https://github.com/MLanguage/mlang.git))
 
-(homepage https://gitlab.inria.fr/verifisc/mlang)
+(homepage https://github.com/MLanguage/mlang)
```
Comment (denismerigoux) [inline dune-project:12]: Thanks!

**Review by denismerigoux — APPROVED** [review]
Looks good to me!

**Comment by mdurero** [comment]
Following yesterday discussion, let’s merge it ASAP so we can at least use it and not wait months again for a third rewrite, test integration, and so on.

**Pull request merged**